test/integration/projects/controls/gcp_projects.rb

```diff
@@ -55,6 +61,12 @@
   'p' => { 'bu1' => prod_bu1_project_floating, 'bu2' => prod_bu2_project_floating }
 }
 
+peering_projects_id = {
```

Comment:
nit: can we add a tests that asserts the peering. 
something with `gcloud compute networks peerings list` perhaps?

Reply:
yeap! I'll add!